gettsim/policy_for_date.py

```diff
@@ -3,27 +3,27 @@
 import numpy as np
 import yaml
 
-from gettsim.benefits.alg2 import e_anr_frei_2005_01
-from gettsim.benefits.alg2 import e_anr_frei_2005_10
-from gettsim.benefits.alg2 import regelberechnung_2011_and_beyond
-from gettsim.benefits.alg2 import regelberechnung_until_2010
-from gettsim.benefits.kiz import calc_kiz_amount_07_2019
-from gettsim.benefits.kiz import calc_kiz_amount_2005
-from gettsim.benefits.unterhaltsvorschuss import uhv_pre_07_2017
-from gettsim.benefits.unterhaltsvorschuss import uhv_since_07_2017
+from gettsim.benefits.arbeitsl_geld_2 import e_anr_frei_2005_01
+from gettsim.benefits.arbeitsl_geld_2 import e_anr_frei_2005_10
+from gettsim.benefits.arbeitsl_geld_2 import regelberechnung_2011_and_beyond
+from gettsim.benefits.arbeitsl_geld_2 import regelberechnung_until_2010
+from gettsim.benefits.kinderzuschlag import calc_kiz_amount_07_2019
+from gettsim.benefits.kinderzuschlag import calc_kiz_amount_2005
+from gettsim.benefits.unterhalt import uhv_pre_07_2017
+from gettsim.benefits.unterhalt import uhv_since_07_2017
 from gettsim.benefits.wohngeld import calc_max_rent_since_2009
 from gettsim.benefits.wohngeld import calc_max_rent_until_2008
 from gettsim.config import ROOT_DIR
 from gettsim.pensions import _rentenwert_from_2018
 from gettsim.pensions import _rentenwert_until_2017
 from gettsim.social_insurance import calc_midi_contributions
 from gettsim.social_insurance import no_midi
-from gettsim.taxes.calc_taxes import no_soli
-from gettsim.taxes.calc_taxes import soli_formula_1991_92
-from gettsim.taxes.calc_taxes import soli_formula_since_1995
-from gettsim.taxes.calc_taxes import tarif
-from gettsim.taxes.kindergeld import kg_eligibility_hours
-from gettsim.taxes.kindergeld import kg_eligibility_wage
+from gettsim.taxes.e_st import st_tarif
+from gettsim.taxes.kindergeld import kindergeld_anspruch_nach_lohn
+from gettsim.taxes.kindergeld import kindergeld_anspruch_nach_stunden
+from gettsim.taxes.soli_st import keine_soli_st
+from gettsim.taxes.soli_st import soli_st_formel_1991_92
+from gettsim.taxes.soli_st import soli_st_formel_seit_1995
 from gettsim.taxes.zve import calc_hhfreib_from2015
```

Comment:
Please add an underscore according to the variable that is calculated (`hh_freib`). Also make such changes elsewhere. Makes it much easier to search & replace later on.

Reply:
I didn't want to touch functions yet. I wanted to postpone when setting up the new interface!

Reply:
[A Foolish Consistency is the Hobgoblin of Little Minds](A Foolish Consistency is the Hobgoblin of Little Minds)